MediaRecorder.html

```diff
@@ -370,18 +397,19 @@
           </dd>
           <dt><dfn><code>resume</code></dfn></dt>
           <dd>
-            When a <code>MediaRecorder</code> object&#8217;s
-            <code>resume()</code> method is invoked, the UA MUST queue a task,
-            using the DOM manipulation task source, that runs the following
-            steps:
+            When a <code>MediaRecorder</code> object&#8217;s <code>resume()
+            </code> method is invoked, the UA MUST queue a task, using the DOM
+            manipulation task source, that runs the following steps:
             <ol>
-              <li>If <code>state</code> is "inactive" throw an
-              <code>InvalidStateError</code> DOMException and terminate these
-              steps. Otherwise:</li>
-              <li>Set <code>state</code> to "recording".</li>
-              <li>Resume (or continue) gathering data into its current
-              Blob.</li>
-              <li>Fire an event named <code>resume</code>.</li>
+              <li>If <a>state</a> is <a href="#dom-recordingstate-inactive">
+              inactive</a> throw an <code>InvalidStateError</code> DOMException
```

Comment:
You only changed the formatting, but this doesn't make sense. Inside a task, what does it mean to throw an exception? In Blink implementation terms, you'd have no `ExceptionState` available.